frontend/src/components/home/mainSection/DatasetsSection.tsx

```diff
@@ -0,0 +1,129 @@
+import Heading from "@components/_shared/Heading";
+import { Button } from "@components/ui/button";
+import {
+  ArrowRightIcon,
+  ArrowUturnRightIcon,
+  BuildingLibraryIcon,
+  ClipboardIcon,
+  GlobeAltIcon,
+  ShieldCheckIcon,
+} from "@heroicons/react/20/solid";
+import { Dataset } from "@portaljs/ckan";
+
+export default function DatasetsSection({
+  datasets,
+}: {
+  datasets: Array<Dataset>;
+}) {
+  const _datasets = [
+    {
+      state: "TDC Formatted",
+      title: "ADB Asian Transport Outlook Database",
+      tags: ["Transport data", "Economic Impact of Transport"],
+      description:
+        "Asian Transport Outlook (ATO) is initiated by the Asian Development Bank (ADB) to strengthen the knowledge base on transport in the Asia-Pacific region. The ATO is developed in support of the planning and delivery of ADB Transport Sector Assistance. The ATO also supports Asian governments in transport policy development and delivery. ADB is working with other interested parties in developing the ATO as an instrument to track the implementation of the Sustainable Development Goals (SDG), the Paris Agreement and other relevant international agreements on sustainable development in the transport sector.",
+      metadata_modified: "2024-02-20T14:00:00Z",
+      organization: "TDC",
+      region: "Worldwide",
+    },
+    {
+      state: "TDC Harmonised",
+      title: "TDC Global Vehicle Registration",
+      tags: ["Vehicle registration", "Transport data"],
+      description:
+        "This dataset and documentation contains detailed information on vehicle registration around the world, a harmonised and up to date transport data set of historical values, 1970 - 2022.",
+      metadata_modified: "2024-02-20T14:00:00Z",
+      organization: "TDC",
+      region: "Worldwide",
+    },
+    {
+      state: "TDC Harmonised",
+      title: "TDC Global Passenger Activity",
+      tags: ["Passenger activity", "Transportation behaviour"],
+      description:
+        "This dataset contains survey data collected from residents of various urban areas, providing insights into transportation behaviors, preferences, and challenges. It can be used to inform the development of sustainable urban mobility solutions and policies.",
+      metadata_modified: "2024-02-20T14:00:00Z",
+      organization: "TDC",
+      region: "Worldwide",
+    },
+  ];
+
+  return (
+    <div className="container py-[96px]">
+      <div className="mx-auto text-center lg:max-w-[640px]">
+        <Heading>Recently added</Heading>
+        <p className="mt-4 text-xl font-normal text-gray-500">
+          Explore available datasets and gain valuable insights into the
+          transportation trends globally
+        </p>
+      </div>
+
+      <div className="mt-16 grid grid-cols-1 gap-[65px] md:grid-cols-2 lg:grid-cols-3">
+        {_datasets.map((dataset, i) => {
```

Comment:
There is a small separator missing i believe
![image](https://github.com/user-attachments/assets/a2949b37-58f4-4e88-80b2-9409afb49089)